src/go/rdctl/cmd/root.go

```diff
@@ -71,7 +71,10 @@ func init() {
 
 func versionCommand(version string, command string) string {
 	if version == "" {
-		return fmt.Sprintf("%s/%s", apiVersion, command)
+		version = apiVersion
+	}
+	if strings.HasPrefix(command, "/") {
+		return fmt.Sprintf("%s%s", version, command)
 	}
 	return fmt.Sprintf("%s/%s", version, command)
```

Comment:
Equivalently,
```suggestion
	return fmt.Sprintf("%s/%s", version, strings.TrimLeft(command, "/"))
```
(But it's not an important change, merging as-is is fine too.)